classwork/05-classwork.qmd

```diff
@@ -0,0 +1,287 @@
+---
+title: "5 - Feature engineering - Classwork"
+subtitle: "Machine learning with tidymodels"
+editor_options: 
+  chunk_output_type: console
+---
+
+We recommend restarting R between each slide deck!
+
+## Case study
+
+```{r}
+library(tidymodels)
+library(ongoal)
+
+tidymodels_prefer()
+ggplot2::theme_set(ggplot2::theme_bw())
+
+str(season_2015)
```

Comment:
I changed this to `glimpse()` in #53

Reply:
Thanks for the heads-up! I've turned it into a suggestion so it can easily be added based on what happens in #53 . Looks nice!